lessons/Algebra2/periodic1-exploring-rides/langs/en-us/index.adoc

```diff
@@ -104,173 +99,238 @@ This worksheet had you create a *new type of data visualization*, called a @voca
+@QandA{
+@Q{When does it make sense to connect the dots on a scatter plot?}
+@A{When the explanatory variable is continuous, like time.}
+}
 
-=== Launch
 
-@right{@image{images/ferris-wheel.png, 150}}Your teacher's seat on the Ferris Wheel can be thought of as a point on the circumference of a rotating circle.
+== Unit Clocks
 
-- The y-value (`altitude`) goes from 5ft to 205ft then back down to 5, then up again, then down again...
-- This pattern of y-values repeats at regular intervals (every 30m)
+=== Overview
+Students begin to reason about points on the circumference of a circle, with their x- and y-coordinates varying as a function of _rotation_.
 
-How do we figure out how far off the ground (y-coordinate) the seat is after some length of time?
+=== Launch
 
-@slidebreak
+@right{@image{images/ferris-wheel.png, 150}}The teacher's seat on the Ferris Wheel can be thought of as a point on the circumference of a rotating circle.
 
-Let's consider the models we've studied:
+- The y-coordinate (`altitude`) goes from 5ft to 205ft then back down to 5, then up again, then down again...
+- This pattern of y-coordinates repeats at regular intervals (every 30m)
 
-- linear, exponential and logarithmic models either keep increasing forever or keep decreasing forever.
-- quadratic relationships grow one way until they reach a maxima or minima _once_, before growing in the other direction.
+@slidebreak 
 
-*None of the tools in our modeling toolkit work for functions that keep repeating, going up and down over time!*
+None of the models we've seen so far will help us predict how far off the ground (y-coordinate) the seat is after a length of time (x-coordinate). Some of them increase or decrease forever (linear, exponential, logarithmic), and others change directions _once_ (quadratic), but not over and over in a cycle!
 
 @slidebreak
 
-This is a problem, because modeling cyclical relationships is _incredibly important_, for everyone from farmers to fishermen to healthcare providers! So many things in nature come in cycles:
-
-- The sun rises and sets every day: @math{\text{sun-height}(\text{time})} is periodic
-- The moon waxes and wanes: @math{\text{moon}(\text{date})} is periodic
-- the tides come in and out each day: @math{\text{tide}(\text{time})} is periodic
-- Tulips tend to bloom in the Spring: @math{\text{tulips}(\text{date})} is periodic
-- People tend to get sick in the winter: @math{\text{flu-cases}(\text{date})} is periodic
-
-In this lesson we'll explore a new class of functions - _@vocab{periodic functions}_ - that we can use to model cyclical relationships like these.
+Modeling cyclical relationships is _incredibly important_, for everyone from farmers to fishermen to healthcare providers! +
+So many things in nature come in cycles:
 
-@strategy{A note on Vocabulary}{
+- The sun rises and sets every day:  @math{\text{sun-height}(\text{time})} is periodic
```

Comment:
@schanzer Noting that we are writing "is periodic" here a few lines before we introduce it. I think I'm ok with that, but not totally sure. The word the we've already introduced would be "cyclical"

Reply:
Adding some small adjustments that work the word in earlier.